packages/docusaurus/bin/utils.mjs

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Please don't create utils files in bin... Just directly inline this in `docusaurus.mjs`.

Reply:
We already have `beforeCli` and may need other similar utils in the future so why not?

Reply:
Thanks for the review, my only reason for creating the utils file is to not pollute the original `docusaurus.mjs`. I added the function there originally but it looked out of place and if any more validation functions are needed in the future it would take away the simplicity and readability of that file. If there are still concerns I am happy to put it back in the main file.

Reply:
`beforeCli` factors out a fair chunk of side-effect code, and semantically speaking it's part of the "CLI process". This file seems to be too trivial, and we also have utility functions inlined in the code, so yeah, you don't have to extract it to a separate module.

Reply:
Fixed